dev/io.openliberty.mcp.internal_fat/build.gradle

```
@@ -18,4 +18,8 @@ configurations.requiredLibs {
 dependencies {
   requiredLibs 'org.json:json:20080701',
                'org.skyscreamer:jsonassert:2.0-rc1'
+}
+/* added below to allow for method parmaters to be available for MCP Tool Server*/
+compileJava {
+    options.compilerArgs << '-parameters'
 }
```

```suggestion
compileJava {
    // Allow us to test getting the name of a tool argument directly from the Java parameter name
    options.compilerArgs << '-parameters'
}
```

We want to remind ourselves why we're doing this, not just what the argument does.